membership-common/src/test/scala/com/gu/memsub/subsv2/services/TestCatalog.scala

```diff
@@ -131,6 +131,13 @@ object TestCatalog {
       Map(contributorChargeId -> Contributor),
       Some(ProductType("type")),
     ),
+    guardianAdLitePrpId -> ProductRatePlan(
+      guardianAdLitePrpId,
+      "Guardian Ad Lite",
+      idForProduct(Product.AdLite),
+      Map(guardianAdLiteChargeId -> GuardianAdLite),
+      Some(ProductType("typeAdLite")),
```

Comment:
if I remember right this is mainly needed for the specific things e.g home delivery days